test/integration/user_can_view_items_from_an_individual_artist_test.rb

```diff
@@ -0,0 +1,18 @@
+require "test_helper"
+
+class UserCanViewItemsFromAnIndividualArtistTest < ActionDispatch::IntegrationTest
```

Comment:
Ignoring this, I don't think the line defining the test class being too long is really an issue worth fixing.